pallets/msa/src/mock.rs

```diff
@@ -67,7 +67,7 @@ impl pallet_schemas::Config for Test {
 }
 
 parameter_types! {
-	pub const MaxPublicKeysPerMsa: u8 = 10;
+	pub const MaxPublicKeysPerMsa: u8 = 255;
```

Comment:
Setting the mocked `MaxPublicKeysPerMsa` to `255` allows the existing unit test `add_key_with_more_than_allowed_should_panic` to hit the overflow case. If an increment on `key_count` would overflow, `Error::<T>::KeyLimitExceeded` is returned.

Reply:
Looks good!